packages/lit-helpers/test/live.test.js

```diff
@@ -0,0 +1,132 @@
+import { expect, html, fixture } from '@open-wc/testing';
+import { spy } from 'sinon';
+import { render } from 'lit-html';
+import { live } from '../src/live.js';
+
+class LitHelpers extends HTMLElement {
```

Comment:
that should go in a before or directly in the function as otherwise it will always be executed even if another test has .only

Reply:
Does that really matter though? I can put it in a before since it's a side effect, but I'm not sure what difference it would make.

Reply:
Added a before for this